sdks/python/apache_beam/io/gcp/bigtableio_it_test.py

```diff
@@ -0,0 +1,189 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+""" Integration test for GCP Bigtable testing."""
+from __future__ import absolute_import
+
+import argparse
+import datetime
+import logging
+import random
+import string
+import time
+import unittest
+from nose.plugins.attrib import attr
+
+import apache_beam as beam
+from apache_beam.metrics.metric import MetricsFilter
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.runners.runner import PipelineState
+from apache_beam.testing.util import assert_that, equal_to
+from apache_beam.transforms.combiners import Count
+
+try:
+  from google.cloud.bigtable import enums, row, column_family, Client
+except ImportError:
+  Client = None
+
+import bigtableio
+
+class GenerateTestRows(beam.PTransform):
+  """ A PTransform to generate dummy rows to write to a Bigtable Table.
+
+  A PTransform that generates a list of `DirectRow` and writes it to a Bigtable Table.
+  """
+  def __init__(self):
+    super(self.__class__, self).__init__()
+    self.beam_options = {'project_id': PROJECT_ID,
+                         'instance_id': INSTANCE_ID,
+                         'table_id': TABLE_ID}
+
+  def _generate(self):
+    for i in range(ROW_COUNT):
+      key = "key_%s" % ('{0:012}'.format(i))
+      test_row = row.DirectRow(row_key=key)
+      value = ''.join(random.choice(LETTERS_AND_DIGITS) for _ in range(CELL_SIZE))
+      for j in range(COLUMN_COUNT):
+        test_row.set_cell(column_family_id=COLUMN_FAMILY_ID,
+                          column=('field%s' % j).encode('utf-8'),
+                          value=value,
+                          timestamp=datetime.datetime.now())
+      yield test_row
+
+  def expand(self, pvalue):
+    return (pvalue
+            | beam.Create(self._generate())
+            | bigtableio.WriteToBigTable(project_id=self.beam_options['project_id'],
+                                         instance_id=self.beam_options['instance_id'],
+                                         table_id=self.beam_options['table_id']))
+
+@unittest.skipIf(Client is None, 'GCP Bigtable dependencies are not installed')
+class BigtableIOTest(unittest.TestCase):
+  """ Bigtable IO Connector Test
+
+  This tests the connector both ways, first writing rows to a new table, then reading them and comparing the counters
+  """
+  def setUp(self):
+    self.result = None
+    self.table = Client(project=PROJECT_ID, admin=True)\
+                    .instance(instance_id=INSTANCE_ID)\
+                    .table(TABLE_ID)
+
+    if not self.table.exists():
+      column_families = {COLUMN_FAMILY_ID: column_family.MaxVersionsGCRule(2)}
+      self.table.create(column_families=column_families)
+      logging.info('Table {} has been created!'.format(TABLE_ID))
+
+  @attr('IT')
+  def test_bigtable_io(self):
+    print 'Project ID: ', PROJECT_ID
+    print 'Instance ID:', INSTANCE_ID
+    print 'Table ID:   ', TABLE_ID
```

Comment:
Maybe avoid printing here, or use `logging.info` instead. If you do want to use print, use `print(...)` as a function